target/linux/ramips/dts/mt7628an_netgear_r6120.dts

```diff
@@ -40,17 +40,18 @@
 			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
 		};
 
-		wlan {
+		wlan2g_green {
 			label = "r6120:green:wlan2g";
 			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
```

Comment:
Please do not mix changes to r6120 with adding R6080.

Have you tested these changes on R6120?

Please separate changes by creating three commits:
1. Rename the dts nodes for 6120
2. create DTS trigger and remove line from 01_leds
3. add R6080

Reply:
> > wlan1 uses the MAC address two above it (e.g. B0:39:56:00:00:02, which appears in the factory partition at 0x8044).
> 
> Should be 0x8004? If you have a valid address there that matches 5 GHz MAC address on stock FW, you can remove the
> 
> ```
> mtd-mac-address = <&factory 0x4>;
> mtd-mac-address-increment = <(2)>;
> ```
> 
> from wifi@0,0. (If mtd-eeprom is 0x8000, the MAC address will be read from 0x8004 automatically.)

Yes it should be 0x8004; will do.

Reply:
Should now be separated as requested. I do not have access to an R6120, so I cannot test.